plugins/source/aws/codegen/recipes/eventbridge.go

```diff
@@ -53,10 +53,112 @@ func EventbridgeResources() []*Resource {
+			Description: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Archive.html",
+			ExtraColumns: append(defaultRegionalColumns, []codegen.ColumnDefinition{{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				Resolver: `resolveArchiveArn`,
+			}}...),
+		},
+		{
+			SubService:  "connections",
+			Struct:      &types.Connection{},
+			Description: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Connection.html",
+			SkipFields:  []string{"ConnectionArn"},
+			ExtraColumns: append(defaultRegionalColumns, []codegen.ColumnDefinition{{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				Resolver: `schema.PathResolver("ConnectionArn")`,
+			},
+			}...),
+		},
+		{
+			SubService:  "event_sources",
+			Struct:      &types.EventSource{},
+			Description: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_EventSource.html",
+			SkipFields:  []string{"Arn"},
+			ExtraColumns: append(defaultRegionalColumns, []codegen.ColumnDefinition{{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				Resolver: `schema.PathResolver("Arn")`,
+			},
+			}...),
+		},
+		{
+			SubService:  "replays",
+			Struct:      &types.Replay{},
+			Description: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Replay.html",
+			ExtraColumns: append(defaultRegionalColumns, []codegen.ColumnDefinition{{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				Resolver: `resolveReplayArn`,
+			},
+			}...),
+		},
+	}
+
+	autogenGlobalResources := []*Resource{
+		{
+			SubService:  "endpoints",
+			Struct:      &types.Endpoint{},
+			Description: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Endpoint.html",
+			SkipFields:  []string{"Arn"},
+			ExtraColumns: append(defaultAccountColumns, []codegen.ColumnDefinition{{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				Resolver: `schema.PathResolver("Arn")`,
+			},
+			}...),
+		},
+	}
+
 	// set default values
 	for _, r := range resources {
 		r.Service = "eventbridge"
 		r.Multiplex = `client.ServiceAccountRegionMultiplexer("events")`
 	}
+
+	for _, r := range autogenResources {
+		r.Service = "eventbridge"
+		r.Multiplex = `client.ServiceAccountRegionMultiplexer("events")`
+		r.ShouldGenerateResolverAndMockTest = true
+		r.ResolverAndMockTestTemplate = "list_resources_1"
+		r.CloudqueryServiceName = "EventBridge"
+	}
+
+	for _, r := range autogenGlobalResources {
+		r.Service = "eventbridge"
+		r.Multiplex = `client.AccountMultiplex`
+		r.ShouldGenerateResolverAndMockTest = true
+		r.ResolverAndMockTestTemplate = "list_resources_1"
+		r.CloudqueryServiceName = "EventBridge"
+	}
```

Comment:
Nit: Probably personal taste, but these are similar enough that I'd keep it all in one list and one for loop, and explicitly mark the ones to be generated using `ShouldGenerateResolverAndMockTest: true`. The multiplex edge case can also be handled by doing:

```
// provide a default multiplexer
if r.Multiplex == "" {
     r.Multiplex = `client.ServiceAccountRegionMultiplexer("events")`
}
```

and then setting multiplex to `client.AccountMultiplex` in the few cases that need it.

Reply:
Done for the boolean - all resources need to explicitly set it to false/true (good, I think).
Left two lists for `globalResources` and `regionalResources`, because I don't think having a "default" is a good fit (is "global" or "regional" a more reasonable default?...).